experimental/traffic-portal/nightwatch/globals/globals.ts

```diff
@@ -382,6 +390,19 @@ const globals = {
 			console.log(`Successfully created Type ${respType.name}`);
 			data.type = respType;
 
+			const profile: RequestProfile = {
+				cdn: 1,
```

Comment:
nit: I would prefer this to use `respCDN.id` so that the tests work regardless of what the environment currently looks like; however, CDN 1 is the `ALL` CDN which is not able to be deleted so this is fine and should always work.